src/main/constants.ts

```diff
@@ -6,10 +6,16 @@
 
 import { app } from "electron";
 import { existsSync, readdirSync, renameSync, rmdirSync } from "fs";
-import { join } from "path";
+import { join, resolve } from "path";
+
+const vesktopDir = resolve(__dirname, "..", "..", "..", "..");
```

Comment:
this seems really hacky and platform dependant

maybe try using something like app.getAppPath() or process.execPath